client-java/src/unit-tests/java/uk/gov/gchq/palisade/client/testing/WsEndpointFilteredResource.java

```diff
@@ -128,22 +128,20 @@ public void onOpen(final String token, final WebSocketSession session) {
         if (!messages.hasNext()) {
             sendComplete(session);
         }
-        LOGGER.debug("onOpen::{}", session.getId());
+        LOGGER.debug("WebSocket Server opened");
     }
 
-    boolean completeSent = false;
-
     /**
      * Called when a new message arrives
      *
      * @param inmsg   The incoming message
      * @param session The web socket session
      */
     @OnMessage
-    public void onMessage(final Message inmsg, final WebSocketSession session) {
-        LOGGER.debug("Recd: {}", inmsg);
+    public void onMessage(final Item inmsg, final WebSocketSession session) {
```

Comment:
i get a "Path variable 'token' is not consumed" micronaut warning here, anything to be concerened over?

Reply:
Ahh. I think I might know what this is, but I'll check.

Reply:
I can't get you you see. I've put an `assert` and a debug log line in `onOpen` and no assert fires as it gets the token from the path. I cannot see that warning in the log. What are you running to get it?

Reply:
its from an intelliJ inspection of the tests, without running them. It might be nothing, just thought it was worth bringing up

Reply:
Ah, there is a replacement on the path in the configuration "%t" which is replaced with the token at run time. Maybe it's not matching that to a path variable in the test server method.